source/seismograph/index.rst

```diff
@@ -0,0 +1,41 @@
+地震仪
+======
+
+
+距离和角度
+----------
+
+震中距
+++++++
+
+地震、球面上任意一点和球心所在的球面圆形称为大圆（*great circle*）。
+大圆由从震中到这一点的小圆弧（*minor arc*）和大圆弧（*major arc*）
+两部分组成。
+
+震中距指震中到地面上任意一点的球面距离，即小圆弧或大圆弧的长度，单位一般
+为度或公里等。除非显示申明，一般默认是小圆弧长度。
```

Comment:
We should mention that 1°=111.2 km, and km is commonly used for local earthquakes, and degree is used for tele earthquakes.